shell/store/index.js

```diff
@@ -615,15 +615,17 @@ export const mutations = {
     state.isRancherInHarvester = neu;
   },
 
-  updateNamespaces(state, { filters, all, getters }) {
+  updateNamespaces(state, opt) {
+    const { filters, all } = opt;
+
     state.namespaceFilters = filters.filter((x) => !!x);
 
     if ( all ) {
       state.allNamespaces = all;
     }
     // Create map that can be used to efficiently check if a
     // resource should be displayed
-    getActiveNamespaces(state, getters);
+    getActiveNamespaces(state, opt.getters || getters);
```

Comment:
we should probably add a comment explaining why the fallback is needed (in case someone removes in the future)